src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content/index.js

```diff
@@ -47,8 +47,8 @@ const TopCountriesWithMoreWebsitesWithMixedContent = () => {
     <>
       <CustomCard data-testid="top-countries-with-more-websites-more-government-with-mixed-content-card">
         <CardHeader
-          title="Countries with more government websites that have mixed content"
-          subheader="These are number of government websites from each country that contain any kind of mixed content."
+          title="Total of government websites that have mixed content"
```

Comment:
nit here and below: just "Total government"

Reply:
Done! thanks for the review